packages/pmml-editor/src/editor/components/Outputs/molecules/OutputFieldEditRow.tsx

```diff
@@ -249,3 +249,5 @@ export const OutputFieldEditRow = (props: OutputFieldEditRowProps) => {
     </section>
   );
 };
+
+export default OutputFieldEditRow;
```

Comment:
I changed a couple of inline exports because components turn out as `Anonymous` in the react inspector, making them difficult to debug